waitext/extensions.go

```diff
@@ -0,0 +1,118 @@
+package waitext
+
+import (
+	"context"
+	"math"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	DefaultInitialInterval     = 500 * time.Millisecond
+	DefaultRandomizationFactor = 0.5
+	DefaultMultiplier          = 1.5
+	DefaultMaxInterval         = 60 * time.Second
```

Comment:
Are this constants used outside of the package? Do you think we could not export them?